src/Microsoft.AspNetCore.Dispatcher/MatcherLoggerExtensions.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Dispatcher
+{
+    internal static class MatcherLoggerExtensions
```

Comment:
This has a bunch of extensions methods that are used in a bunch of classes, so you can give this the name `LoggerExtensions`